ui/components/show/showCards/ShowCards.tsx

```diff
@@ -12,72 +11,74 @@ import CardImgContainer from './ImgContainer/CardImgContainer';
 import TagsContainer from '../../layout/cardElements/TagsContainer/TagsContainer';
 import ContentSection from '../../layout/ContentSection/contentSection';
 
-import styles from "./ShowCards.module.scss";
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper-bundle.min.css';
 
-interface ShowCardsProps{
-    shows: Show[]
-  }
+import styles from "./ShowCards.module.scss";
 
+interface ShowCardsProps {
+  shows: Show[];
+  singleRow?: boolean;  // New prop to control single row display
+}
 
-function ShowCards({shows}:ShowCardsProps) {
+function ShowCards({ shows, singleRow = false }: ShowCardsProps) {
+  if (!shows) {
+    return null;
+  }
 
+  const cardContent = (show: Show) => (
+    <Card key={show.id}>
+      <div>
+        <Heading>
+          <div className={styles.eperDate}>
```

Comment:
```suggestion
          <div className={styles.epDate}>
```
This is a typo I made a long time ago.